specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/virtualNetwork.json

```diff
@@ -1376,6 +1431,50 @@
         }
       },
       "description": "Usage strings container."
+    },
+    "PrepareNetworkPoliciesRequest": {
+      "properties": {
+        "serviceName": {
+          "type": "string",
+          "description": "The name of the service for which subnet is being preparation for."
+        },
+        "resourceGroupName": {
+          "type": "string",
+          "description": "The name of the resource group where the Network Intent Policy will be stored."
+        },
+        "networkIntentPolicyConfigurations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkIntentPolicyConfiguration"
+          },
+          "description": "A list of NetworkIntentPolicyConfiguration."
+        }
+      }
+    },
+    "NetworkIntentPolicyConfiguration": {
+      "properties": {
+        "networkIntentPolicyName": {
+          "type": "string",
+          "description": "The name of the Network Intent Policy for storing in target subscripton."
```

Comment:
subscripton -> subscription

Reply:
Thanks for catching that. I have taken care of it in latest update